src/domains/user/user.model.ts

```diff
@@ -0,0 +1,38 @@
+import { DocumentData } from 'firebase/firestore';
+import { UserType } from '../../interfaces';
+
+/**
+ * Base interface for user information
+ */
+export interface IUserInfo {
+  roarUid?: string;
+  assessmentUid: string;
+  assessmentPid?: string;
+  userType?: UserType;
+  userMetadata?: { [key: string]: unknown };
+  testData?: boolean;
+  demoData?: boolean;
+  offlineEnabled?: boolean;
+  offlineTasks?: string[];
+  offlineAdministrations?: string[];
```

Comment:
`offlineEnabled`, `offlineTasks`, and `offlineAdministrations` is cruft from a previously attempted enhancement please remove this and all references to it elsewhere.